src/hierarchyFinder.js

```diff
@@ -0,0 +1,62 @@
+var _ = require('lodash');
+
+module.exports = {};
+
+var hasName = function(r) {
+  return r.name;
```

Comment:
Would we ever want a record where the name is the empty string (`""`) to pass this function? The number zero, `false` and the empty string will all evaluate to false in Javascript conditionals.